### ISSUE TITLE
Windows: Failed tests in IndexBaseAndGitRepoJobTest

### ISSUE THREAD

**Issue by quasarea** [boyter/searchcode-server#94]
Failed tests:
  IndexBaseAndGitRepoJobTest.testCloneSucess:48
  IndexBaseAndGitRepoJobTest.testIndexSucess:39

Tests run: 285, Failures: 2, Errors: 0, Skipped: 0

It seems like it maybe related to linux path notation of "\tmp\" as I run it on windows.

**Comment by boyter** [comment]
Yep that looks likely. Will attempt to get a Windows box setup so I can resolve it.

**Comment by quasarea** [comment]
maybe also worth mentioning that you need http://gnuwin32.sourceforge.net/packages/coreutils.htm in order to process post-compile processing like:
```
[localhost] local: rm -rf ./release/*
[localhost] local: cp -R ./target/dependency-jars ./release/
[localhost] local: cp ./target/*.jar ./release/
[localhost] local: cp ./searchcode.properties.example ./release/searchcode.properties
[localhost] local: cp ./quartz.properties ./release/quartz.properties
[localhost] local: cp ./searchcode.sqlite.empty ./release/searchcode.sqlite
[localhost] local: cp ./README.md ./release/
[localhost] local: chmod +x ./searchcode-server.sh
[localhost] local: cp ./searchcode-server.sh ./release/
[localhost] local: cp ./searchcode-server.bat ./release/
[localhost] local: mkdir ./release/logs
```
unless there is a way of making it os dependent.

**Comment by boyter** [comment]
https://github.com/boyter/searchcode-server/issues/95

**Comment by boyter** [comment]
This should be resolved in dfb6763c559d10b9f1b49f642405a84c0ed18a5f as it now uses the java.io.tmpdir property to build the temp directory.
